articles/getting-started-with-jwt-using-angular8-and-nodejs/index.md

```diff
@@ -0,0 +1,362 @@
+Unauthorized was displayed as expected. Next will be to test it with the token I
+first obtained from the token/sign route as show below:
+
+```javascript
+app.get('/path1', (req, res) => {
+    res.status(200)
+        .json({
+            "success": true,
+            "msg": "Secret Access Granted"
+        });
+});
+```
+
+On adding the Bearer Token, the success message is displayed as shown below:
+
+![success message](/articles/getting-started-with-jwt-using-angular8-and-nodejs/success-token.png)
+
+Heading back to Angular, I will create a new component home by running the
+following command:
+
+```javascript
+ng generate component home
+```
+
+Modifying my home.component.ts located at *src/app/home/home.component.ts*  as
+below:
+
+```javascript
+signIn() {
+    this.http.get(this.API_URL + '/token/sign')
+      .subscribe(
+        (res) => {
+          console.log(res);
+          if (res['token']) {
+            localStorage.setItem('token', res['token']);
+          }
+        },
+        (err) => {
+          console.log(err);
+        }
+      );    
+  }
+
+  getPath() {
+    this.http.get(this.API_URL + '/path1')    
+      .subscribe(
+        (res) => {
+          console.log(res);
+        },
+        (err) => {
+          console.log(err);
+        }
+      );       
+  }
+
+```
+
+The *signIn()* function in the above code requests a token a stores it into local
+storage, while the second *getPath()* function is for requesting the path.
+
+Coding a simple HTML page to test the two functions above, I will browse to */src/app/home/home.component.html* and code the below: 
+
+```HTML
+<h3>JSON Web Test</h3>
+
+<button (click)="signIn()">Sign In</button>
+<button (click)="getPath()">Get Path 1</button>
+```
+
+To run an Angular project on the browser, I will browse the project folder
+using **cmd** and run the command *ng serve*. The command will compile the whole Angular project and launch the page on the browser. Press F12 on the keyboard to open developer tools console window and then click on the **Sign In** button to get a token as explained above, then click on the **Get Path1** Button to access the path as shown below:
+
+![web test success](/articles/getting-started-with-jwt-using-angular8-and-nodejs/webtest-success.png)
+
+To lose the local storage token, I will refresh the browser, and then I will try
+to access the path as shown below:
+
+![web test error](/articles/getting-started-with-jwt-using-angular8-and-nodejs/webtest-error.png)
+
+As it is now, we have successfully gotten a 401 Unauthorized, meaning the
+application is now secure.
+
+## Conclusion
+
+Using JSON Web Tokens, the app is now secure since the services and any
+communication between the server and the app is also secure. This guide has also
+helped the reader implement the JSON Web Token in Angular 11 and Node.js. This
+contribution will help anyone to secure their applications and make them ready
+for production.
```

Comment:
Write a sentence in one line instead of breaking it down over several lines. Please go through all your paragraphs  fixing this.